src/scores.js

```diff
@@ -1,32 +1,31 @@
-const scores = [
-  {
-    userName: 'Phelian',
-    score: 80,
-  },
-  {
-    userName: 'John',
-    score: 70,
-  },
-  {
-    userName: 'Achebe',
-    score: 20,
-  },
-  {
-    userName: 'Wole',
-    score: 86,
-  },
-  {
-    userName: 'Dupe',
-    score: 34,
-  },
-  {
-    userName: 'Tito',
-    score: 57,
-  },
-  {
-    userName: 'Jumoke',
-    score: 96,
-  },
-];
+import addScoresToList from './addcores.js';
 
-export default scores;
\ No newline at end of file
+const baseUrl = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/QH8RZRLzI8I3QLPQbGgT/scores/';
+
+const postScores = async () => {
+  const newScore = {
+    user: document.getElementById('user-name').value,
+    score: document.getElementById('user-score').value,
+  };
+
+  (await fetch(baseUrl, {
+    method: 'POST',
+    headers: {
+      'Content-type': 'application/json',
+    },
+    body: JSON.stringify(newScore),
+  })).json();
+};
+
+function displaySores(scores) {
```

Comment:
- [ ] As it is a requirement for this project and best practice, please kindly use `arrow functions` not function key word. 
![phe](https://user-images.githubusercontent.com/65852621/141174015-ea9aa172-f58e-45ce-91bd-d82da5afadda.png)

Reply:
Thanks for the review, I'll implement that